src/test/java/com/triangledeterminer/controller/TriangleControllerTest.java

```diff
@@ -0,0 +1,43 @@
+package com.triangledeterminer.controller;
+
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.triangledeterminer.model.Triangle;
+
+@SpringBootTest
+@AutoConfigureMockMvc
+class TriangleControllerTest {
+	private static final ObjectMapper mapper = new ObjectMapper();
+	private Triangle triangle;
+
+	@Autowired
+	private MockMvc mockMVC;
+	
+	@BeforeEach
+	public void setUp() {
+		triangle = new Triangle();
+	}
+
+	@Test
+	void testGivesOkWhenControllersListensToHTTPRequest() throws Exception {
```

Comment:
We should add more negative test cases in the future

Reply:
Ok, I will take care in future.